portal-api/api-tests/v1/users/login-with-sign-in-link.api-test.js

```diff
@@ -36,7 +36,8 @@ describe('POST /users/:userId/sign-in-link/:signInToken/login', () => {
   let partiallyLoggedInUserId;
   let partiallyLoggedInUserToken;
 
-  const login = ({ userId, signInToken, userToken }) => as({ token: userToken }).post().to(`/v1/users/${userId}/sign-in-link/${signInToken}/login`);
+  const loginWithSignInLink = ({ userId, signInToken, userToken }) =>
+    as({ token: userToken }).post().to(`/v1/users/${userId}/sign-in-link/${signInToken}/login`);
```

Comment:
More for my own understanding, what does "sign in link" mean in this context? Just a token?

Finding the naming "login with sign in link" confusing because "login" and "sign in" mean the same thing.